lib/error_tracker.ex

```diff
@@ -107,6 +107,11 @@ defmodule ErrorTracker do
   * A `{kind, exception}` tuple in which case the information is converted to
   an Elixir exception (if possible) and stored.
   """
+
+  @type exception :: Exception.t() | {:error, any()} | {Exception.non_error_kind(), any()}
+  @type stack_trace :: [{atom(), fun(), integer(), Keyword.t()}]
+
+  @spec report(exception(), stack_trace()) :: Occurrence.t() | :noop
```

Comment:
Since we are already using a few types from `Exception` we may do the same as well for the stacktrace:

```suggestion
  @spec report(exception(), Exception.stacktrace()) :: Occurrence.t() | :noop
```

Reply:
Also, when compiling the docs they don't show the spec for the `report/3` function. I believe that's because the spec is missing the third argument.

```suggestion
  @spec report(exception(), Exception.stacktrace(), context()) :: Occurrence.t() | :noop
```